Source/engine/animationinfo.cpp

```diff
@@ -170,18 +171,35 @@ void AnimationInfo::ChangeAnimationData(CelSprite *pCelSprite, int numberOfFrame
 	DelayLen = delayLen;
 }
 
-void AnimationInfo::ProcessAnimation()
+void AnimationInfo::ProcessAnimation(bool reverseAnimation /*= false*/, bool dontProgressAnimation /*= false*/)
 {
 	DelayCounter++;
+	if (dontProgressAnimation)
+		return;

```

Comment:
It's a bit hard to see if the logic behind this is proper with out any usage, maybe wait with adding `dontProgressAnimation` until it has a use case.